SPECS-EXTENDED/python-curio/python-curio.spec

```diff
@@ -29,11 +29,11 @@ many of its ideas have influenced other libraries with similar functionality.
 Summary:        Building blocks for performing concurrent I/O
 Name:           python-%{pkgname}
 Version:        1.4
-Release:        3%{?dist}
+Release:        4%{?dist}
 License:        BSD
 URL:            https://github.com/dabeaz/curio
 #Source0:       https://files.pythonhosted.org/packages/source/c/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
```

Comment:
Not needed anymore and removing these makes it easier to write tooling.;)

```suggestion
```

Reply:
Removed.